pypechain/test/errors/types/ErrorsContract.py

```diff
@@ -383,15 +382,11 @@ class ErrorsContract(Contract):
     )
 
     def __init__(self, address: ChecksumAddress | None = None) -> None:
-        try:
-            # Initialize parent Contract class
-            super().__init__(address=address)
-            self.functions = ErrorsContractFunctions(errors_abi, self.w3, address)  # type: ignore
+        # Initialize parent Contract class
```

Comment:
what is the motivation behind removing the try/except?

Reply:
Honestly, I don't know the original intent, but the exception that was being caught is no longer exported in web3py

Reply:
Chatted offline, this exception doesn't get thrown on init, but when the fallback function tries to get accessed. Keeping this try/catch out.